src/include/expression/star_expression.h

```diff
@@ -38,7 +38,7 @@ class StarExpression : public AbstractExpression {
   virtual void Accept(SqlNodeVisitor *v) { v->Visit(this); }
 
  protected:
-  StarExpression(const FunctionExpression &other) : AbstractExpression(other) {}
+  StarExpression(const AbstractExpression &other) : AbstractExpression(other) {}
```

Comment:
If I remember correctly, this old version reported the following error when I was adding support for star expressions: "error: no matching function for call to ‘peloton::expression::AbstractExpression::AbstractExpression(const peloton::expression::FunctionExpression&)’
"